data/campaigns/World_Conquest/gui/help_dialog.cfg

```diff
@@ -391,11 +391,13 @@
                                                 [/row]
                                                 [row]
                                                     [column]
+                                                    	  # wmllint: markcheck off	
```

Comment:
Why is this markcheck off? Shouldn't it be `"<b>" + _"Recruits" + ":</b>"` or some such?